CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+master
+------
+- Remove print proxy "xml2pdf" (#1889)
```

Comment:
better use a fully qualified URL for that. Tools other than GitHub.com wont be able to solve the link otherwise